static/bidder-info/progx.yaml

```diff
@@ -0,0 +1,11 @@
+aliasOf: "vidazoo"
+endpoint: "https://exchange.programmaticx.ai/openrtb/"
+maintainer:
+  email: "pxteam@programmaticx.ai"
+gvlVendorID: 1344
+userSync:
+  iframe:
+    url: https://sync.tagoras.io/api/user/html/685297194d85991a5e6e36dd?pbs=true&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}
+    userMacro: ${userId}
+openrtb:
+  version: 2.6
```

Comment:
It is most appropriate to inherit this value from the parent adapter. Please remove the declaration of OpenRTB 2.6.